public/index.html

```diff
@@ -8,6 +8,7 @@
     <script src="https://cdnjs.cloudflare.com/ajax/libs/react/0.12.2/JSXTransformer.js"></script>
     <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/2.1.1/jquery.min.js"></script>
     <script src="https://cdnjs.cloudflare.com/ajax/libs/showdown/0.3.1/showdown.min.js"></script>
+    <!--<script src='js/tutorial1.js'></script>-->
```

Comment:
nit: we'll want `type="text/jsx"` so that if this is uncommented, the JSX will actually get transformed. Otherwise you'll just hit parse errors.